docs/deploy-from-github.md

```diff
@@ -0,0 +1,19 @@
+---
+id: deploy-from-github
+title: Deploy from GitHub
+sidebar_label: Deploy from GitHub
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Nimbella gives you the ability to deploy your applications and serverless functions that are on your GitHub.
+
+The process is very similiar to how you'd normally deploy your code. You start by running `nim project deploy` in your command prompt. But instead providing the local folder path, you provide the link for GitHub.
+
+To test what it's like to deploy an app from GitHub, run this command to deploy the `visits` demo app.
```

Comment:
Before giving this example, we should show what the format is for specifying the github repo. I think the existing nim docs cover this though: https://apigcp.nimbella.io/downloads/nim/nim.html#deploying-directly-from-github